### ISSUE TITLE
EspalomaTemplateGenerator cannot properly load espaloma models trained with dropouts and batch normalization 

### ISSUE THREAD

**Issue by kntkb** [openmm/openmmforcefields#263]
We need to call model.eval() after loading the espaloma model in [generators/template_generators.py#L1580](https://github.com/openmm/openmmforcefields/blob/main/openmmforcefields/generators/template_generators.py#L1580) when loading a model trained with dropouts and batch normalization. It will yield inconsistent inference results if model.eval() is not called.

Reference:
https://pytorch.org/tutorials/beginner/saving_loading_models.html
